websites/H/HentaiZ/presence.ts

```diff
@@ -37,7 +36,7 @@ presence.on("UpdateData", async () => {
     browsingTimestamp = Math.floor(Date.now() / 1000);
   }
 
-  if (!playback) {
+  if (!document.getElementsByClassName("mr-1")[0].querySelector("h2")) {
```

Comment:
Is this basically just

```suggestion
  if (!document.querySelector(".mr-1 h2")) {
```

Reply:
Will test ASAP

Reply:
Yes it is

Reply:
Yes it is, will fix ASAP